mava/wrappers/jumanji.py

```diff
@@ -217,3 +219,101 @@ def observation_spec(self) -> specs.Spec[Union[Observation, ObservationGlobalSta
             )
 
         return spec
+
+
+class CleanerWrapper(MultiAgentWrapper):
+    """Multi-agent wrapper for the Cleaner environment.
+
+    Do not use the AgentID wrapper with this env, it has implicit agent IDs.
+    """
+
+    def __init__(self, env: Cleaner, has_global_state: bool = False):
+        super().__init__(env)
```

Comment:
Just so we have some type hints

```suggestion
        super().__init__(env)
        self._env: Cleaner
        
```